src/source/Ice/IceAgent.h

```diff
@@ -26,8 +26,8 @@ extern "C" {
 /* Control the calling rate of iceCandidateGatheringTimerTask. Can affect STUN TURN candidate gathering time */
 #define KVS_ICE_GATHER_CANDIDATE_TIMER_POLLING_INTERVAL                 50 * HUNDREDS_OF_NANOS_IN_A_MILLISECOND
 
-/* Should see at least one keep alive within this period */
-#define KVS_ICE_ENTER_STATE_DISCONNECTION_GRACE_PERIOD                  2 * KVS_ICE_SEND_KEEP_ALIVE_INTERVAL
+/* ICE should've received at least one keep alive within this period. Since keep alives are send every 15s */
+#define KVS_ICE_ENTER_STATE_DISCONNECTION_GRACE_PERIOD                  30 * HUNDREDS_OF_NANOS_IN_A_SECOND
```

Comment:
I don't understand this change.

Reply:
it should be 2 * KVS_ICE_SEND_KEEP_ALIVE_INTERVAL. changed back.